src/concepts/segments.md

```diff
@@ -0,0 +1,134 @@
+# Segments
+
+A segment is a group of clients who fit a set of criteria at a point in time. 
+The set of criteria itself can also be referred to as a "segment".
+
+Typically you'll use segments to gain more insight into what is going on, by asking 
+"Regarding the thing I'm interested in, 
+do users in different segments behave differently, 
+and what insights does this give me into the users and the product?" 
+For example, "In this experiment, how do new users react to this feature, 
+and how does this differ from established users?". 
+Or "DAU moved dramatically - 
+is this restricted to users in this particular country (i.e. a segment) 
+where there's an event happening, 
+or is it global and therefore not solely due to that event?"
```

Comment:
A little pedantic, but the phenomenon being restricted to a particular country doesn't necessarily tell us what caused the phenomenon.  I'd consider rephrasing to avoid promoting this kind of thinking.

Reply:
Added "which would raise our suspicions", which signals the level of certainty that people should take from such a discovery - sound good?

Reply:
Looks like you edited further?  In any case, I'm fine with whatever you think is appropriate.

Reply:
There's no "outdated" badge on this one because you commented on line 15 then I edited line 14, but yes I updated the source adding "which would raise our suspicions" to line 14